src/main/java/nextstep/subway/applicaion/LineService.java

```diff
@@ -18,12 +21,28 @@ public LineService(LineRepository lineRepository) {
 
     public LineResponse saveLine(LineRequest request) {
         Line line = lineRepository.save(new Line(request.getName(), request.getColor()));
-        return new LineResponse(
-                line.getId(),
-                line.getName(),
-                line.getColor(),
-                line.getCreatedDate(),
-                line.getModifiedDate()
-        );
+        return LineResponse.of(line);
+    }
+
+    public List<LineResponse> findAllLines() {
+        return lineRepository.findAll().stream()
+                .map(LineResponse::of)
+                .collect(Collectors.toList());
+    }
+
+    public LineResponse findLine(Long id) {
+        Line line = lineRepository.findById(id).orElseThrow(() -> new RuntimeException());
+        return LineResponse.of(line);
     }
+
+    public LineResponse modifyLine(Long id, LineRequest lineRequest) {
+        Line line = lineRequest.toEntity(id);
+        Line modifiedLine = lineRepository.save(line);
+        return LineResponse.of(modifiedLine);
+    }
+
+    public void deleteLineById(Long id) {
```

Comment:
```suggestion
    public void delete(Long id) {
```

저장, 수정할 때는 ById를 붙이지 않았으니 동일한 규칙을 가져가는게 좋겠네요.
그리고 메서드명에 Line을 표시하지 않아도 LineService이니 충분히 의미가 전달되지 않을까요?